src/Microsoft.Azure.ServiceBus/Core/IInnerSender.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus.Core
+{
+    using System.Threading.Tasks;
+
+    internal interface IInnerSender
+    {
+        MessageSender InnerSender { get; }
+
+        MessageSender CreateMessageSender();
```

Comment:
True. It was kept for extensibility purposes in the first place. Maybe someday if you ever need to expand it to beyond AMQP. But given the future plans, I dont' see it happening anytime soon. So removing it.